core/includes/bootstrap.inc

```diff
@@ -3702,7 +3702,7 @@ function language_default() {
  * - http://example.com/node/306 returns "node/306".
  * - http://example.com/backdropfolder/node/306 returns "node/306" while
  *   base_path() returns "/backdropfolder/".
- * - http://example.com/path/alias (which is a path alias for node/306) returns
+ * - http://example.com/path/alias (which is a URL alias for node/306) returns
```

Comment:
Wondering if we should also change the example to `http://example.com/url/alias`, and the "path/alias" in the following line to "url/alias" 🤔

Reply:
That makes sense, but that also triggers the thought: `http://example.com/url-alias` seems like a more "alias-like" URL, since they are very often hyphenated versions of the node title. Would that be a better example? Or perhaps, `http://example.com/post/url-alias`?

Reply:
I like your second suggestion @bugfolder but it doesn't fit on one line. So let's use the first one:

```suggestion
 * - http://example.com/url-alias (which is a URL alias for node/306) returns
```